### ISSUE TITLE
is configuring ty with `ty.json` supposed to work yet?

### ISSUE THREAD

**Issue by DetachHead** [astral-sh/ty#302]
### Question

(apologies if it's too early for me to be opening issues on this project. i'm just very excited about ty and eager to start playing around with it)

i can see in [the playground](https://play.ty.dev/2a3c501f-73f6-4c6d-a884-eefec2d42935) that ty is configured with a `ty.json` file, however this does not seem to work when running `ty check` locally:

```jsonc
// ./ty.json

{
  "rules": {
    "invalid-type-form": "ignore"
  }
}
```
```py
# ./foo.py

foo: []
```

```
> ty check foo.py
WARN ty is pre-release software and not ready for production use. Expect to encounter bugs, missing features, and fatal errors.
error[invalid-type-form]: List literals are not allowed in this context in a type expression
 --> foo.py:1:6
  |
1 | foo: []
  |      ^^
  |
info: `invalid-type-form` is enabled by default

Found 1 diagnostic
```

### Version

0.0.0-alpha.8 (0474b40e1 2025-05-09)

**Comment by DetachHead** [comment]
oh it's `ty.toml` instead. my bad

**Comment by carljm** [comment]
Yeah we should maybe find some way to make this clearer in the playground. It's JSON in the playground just because dealing with JSON in JS is much easier than dealing with TOML :)
